CMakeLists.txt

```diff
@@ -25,129 +30,35 @@ set( IODACONV_PYLINT_CFG_DIR ${CMAKE_CURRENT_SOURCE_DIR} )
 set( PYIODACONV_BUILD_LIBDIR   ${CMAKE_BINARY_DIR}/lib/pyiodaconv )
 set( PYIODACONV_INSTALL_LIBDIR ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/pyiodaconv )
 
-
 ## Dependencies
 
-# Optional: gnssro converter dependencies
-# Either jedicmake or ecbuild are required for FindNetCDF.cmake
-find_package(jedicmake QUIET)
-if(jedicmake_FOUND)
-    message(STATUS "Found: jedicmake")
-else()
-    message(STATUS "NOT Found: jedicmake")
-    find_package(ecbuild QUIET)
-    if( NOT ecbuild_FOUND OR NOT EXISTS ${ecbuild_MACROS_DIR})
-        message(STATUS "NOT Found: ecbuild")
-        message(WARNING "Neither jedicmake nor ecbuild was found.")
-    else()
-        message(STATUS "Found: ecbuild")
-    endif()
-    list(APPEND CMAKE_MODULE_PATH ${ecbuild_MACROS_DIR})
-    include(ecbuild_log) #Required for ecbuild/FindNetCDF.cmake
-endif()
+ecbuild_find_python( REQUIRED VERSION 3.7 )
```

Comment:
Looks like Travis-CI build is failing because it has Python 3.6.9 which doesn't meet this requirement. Would it work to require any python 3 version (`REQUIRED VERSION 3`), or do we need to tell Travis-CI to use a newer python version?

Reply:
we should roll this back to `3`.
That should suffice.